v2/thirdpartyemailpassword/common-customizations/embed-sign-in-up-form.mdx

```diff
@@ -210,6 +393,292 @@ function MyLandingPage() {
 </TabItem>
 </FrontendPreBuiltUITabs>
 
+:::important
+In the above case, the `SignInAndUp` component gets unmounted immediately upon user login. Consequently, SuperTokens cannot invoke `getRedirectionURL` (if configured) in this situation. Refer to the next case if you need conditional rendering along with redirection.
+:::
+
+### If you want to conditionally render the Auth Widget with redirection
```

Comment:
Not sure when i want to do that.. like if i want redirection, why would i want to conditionally render the login widget?